lib/msal-common/src/exports-common.ts

```diff
@@ -153,7 +153,62 @@ export {
     createClientConfigurationError,
 } from "./error/ClientConfigurationError.js";
 export {
-    Constants,
+    LIBRARY_NAME,
+    SKU,
+    CACHE_PREFIX,
+    DEFAULT_AUTHORITY,
+    DEFAULT_AUTHORITY_HOST,
+    DEFAULT_COMMON_TENANT,
+    ADFS,
+    DSTS,
+    AAD_INSTANCE_DISCOVERY_ENDPT,
+    CIAM_AUTH_URL,
+    AAD_TENANT_DOMAIN_SUFFIX,
+    RESOURCE_DELIM,
+    NO_ACCOUNT,
+    CLAIMS,
+    CONSUMER_UTID,
+    OPENID_SCOPE,
+    PROFILE_SCOPE,
+    OFFLINE_ACCESS_SCOPE,
+    EMAIL_SCOPE,
+    CODE_GRANT_TYPE,
+    RT_GRANT_TYPE,
+    S256_CODE_CHALLENGE_METHOD,
+    URL_FORM_CONTENT_TYPE,
+    AUTHORIZATION_PENDING,
+    NOT_DEFINED,
+    EMPTY_STRING,
+    NOT_APPLICABLE,
+    NOT_AVAILABLE,
+    FORWARD_SLASH,
+    IMDS_ENDPOINT,
+    IMDS_VERSION,
+    IMDS_TIMEOUT,
+    AZURE_REGION_AUTO_DISCOVER_FLAG,
+    REGIONAL_AUTH_PUBLIC_CLOUD_SUFFIX,
+    KNOWN_PUBLIC_CLOUDS,
+    SHR_NONCE_VALIDITY,
+    INVALID_INSTANCE,
+    HTTP_SUCCESS,
+    HTTP_SUCCESS_RANGE_START,
+    HTTP_SUCCESS_RANGE_END,
+    HTTP_REDIRECT,
+    HTTP_CLIENT_ERROR,
+    HTTP_CLIENT_ERROR_RANGE_START,
+    HTTP_BAD_REQUEST,
+    HTTP_UNAUTHORIZED,
+    HTTP_NOT_FOUND,
+    HTTP_REQUEST_TIMEOUT,
+    HTTP_GONE,
+    HTTP_TOO_MANY_REQUESTS,
+    HTTP_CLIENT_ERROR_RANGE_END,
+    HTTP_SERVER_ERROR,
+    HTTP_SERVER_ERROR_RANGE_START,
+    HTTP_SERVICE_UNAVAILABLE,
+    HTTP_GATEWAY_TIMEOUT,
+    HTTP_SERVER_ERROR_RANGE_END,
+    HTTP_MULTI_SIDED_ERROR,
```

Comment:
Similarly you can `export * as Constants from "./utils/Constants.js"` so we don't need to individually export every single item each time we add new constants.

Reply:
Done